lib/resources/r4/related_person.yaml

```diff
@@ -75,4 +76,309 @@ fields:
+        <ul>
+          <li> Must have a `use` of 'official'. </li>
+          <li> Cannot contain the `text` field. </li>
+          <li> Must contain either the `given` or `family` fields. </li>
+          <li> Up to two `given` names may be provided. If additional given names are required they must be provided in the second value as a single string separated by blank spaces. </li>
+          <li> Only one `prefix` and one `suffix` may be provided. </li>
+          <li> `period.end` is not supported. </li>
+          <li> When specifying a `period`, `period.start` must include a time component with a timezone. </li>
+        </ul>
+      </li>
+    </ul>
+  url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.name
+
+- name: Related Person Encounter
+  required: 'Yes'
+  type: Extension
+  description: A reference to an Encounter.
+  action: create
+  note: |
+    <ul>
+      <li>The <code>valueReference.reference</code> must be an Encounter reference.</li>
+    </ul>
+  example: |
+    {
+      "extension": [
+        {
+          "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter",
+          "valueReference": {
+            "reference": "Encounter/97939518"
+          }
+        }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter?_format=json
+
+- name: Relationship Level
+  required: 'No'
+  type: Extension
+  description: The resource's relationship to either the patient or encounter level.
+  action: create
+  note: |
+    <ul>
+      <li>If provided, the <code>valueCodeableConcept.coding[0].code</code> must be `Encounter`.</li>
+    </ul>
+  example: |
+    {
+      "extension": [
+        {
+          "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/resource-types",
+                "code": "Encounter",
+              }
+            ]
+          }
+        }
+      ]
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level?_format=json
+
+- name: identifier
+  required: 'No'
+  type: List of Identifier
+  description: The identifiers for the related person.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "urn:oid:5.5.5.5.5.5",
+          "value": "&lt;UNIQUE IDENTIFIER&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Subsequent identifiers may be provided, but have the following constraints:
```

Comment:
why are we saying subsequent identifiers here? Aren't the constraints the same for the first identifier as well?

Reply:
updated here; https://github.com/cerner/fhir.cerner.com/pull/527/commits/7d6aadabc274f0e4e4a9ea1f037e6532c0c99abd#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R244-R246